dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpSuccessfulDeleteHistory.scala

```diff
@@ -0,0 +1,48 @@
+package com.wavesplatform.dex.api.http.entities
+
+import com.wavesplatform.dex.domain.order.Order
+import io.swagger.annotations.{ApiModel, ApiModelProperty}
+import play.api.libs.json.{Json, OFormat}
+
+
+@ApiModel(
+  description = "Successful deletion message. Can be HttpSuccessfulDeleteHistory",
+  subTypes = Array(
+    classOf[HttpSuccessfulSingleCancel],
+    classOf[HttpSuccessfulBatchCancel]
```

Comment:
It looks like the `subTypes` are incorrect. Do they?

Reply:
incorrect, yes